internal/daysteps/daysteps.go

```diff
@@ -20,24 +20,39 @@ const (
 func parsePackage(data string) (int, time.Duration, error) {
 
 	parts := strings.SplitN(data, ",", 2)
+
+	if data == "" {
+		log.Println("пустая строка")
+		return 0, 0, errors.New("invalid format")
+	}
+
 	if len(parts) != 2 {
 		log.Println("неверный формат строки")
-		return 0, 0, errors.New("неверный формат данных шагов и времени")
-	} else if data == "" {
-		log.Println("неверный формат строки")
-		return 0, 0, errors.New("пустая строка")
+		return 0, 0, errors.New("invalid format")
 	}
 
 	steps, err := strconv.Atoi(parts[0])
-	if steps <= 0 || err != nil {
-		log.Println(err)
-		return 0, 0, errors.New("недостаточно шагов")
+
+	if steps <= 0 {
+		log.Printf("нулевое или отрицательное значение шагов:%v", err)
+		return 0, 0, errors.New("conversion error steps: negative or zero value ")
+	}
+
+	if err != nil {
+		log.Printf("ошибка при расчёте шагов: %v", err)
+		return 0, 0, errors.New("conversion error steps")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```